README.md

```diff
@@ -19,99 +19,54 @@ You can watch videos of the system being developed on YouTube:
+* [Libraries](Userland/Libraries/) for everything from cryptography to OpenGL, audio, JavaScript, GUI, playing chess, ...
+* Support for many common and uncommon file formats (PNG, JPEG, GIF, MP3, WAV, FLAC, ZIP, TAR, PDF, QOI, Gemini, ...)
+* Unified style and design philosophy, flexible theming system, [custom (bitmap and vector) fonts](https://fonts.serenityos.net/font-family)
+* [Games](Userland/Games/) (Solitaire, Minesweeper, 2048, chess, Conway's Game of Life, ...) and [demos](Userland/Demos/) (CatDog, Starfield, Eyes, mandelbrot set, WidgetGallery, ...)
+* Every-day GUI programs and utilities (Spreadsheet with JavaScript, TextEditor, Terminal, PixelPaint, various multimedia viewers and players, Mail, Assistant, Calculator, ...)
+
+... and all of the above are right in this repository, no extra dependencies, built from-scratch by us :^)
+
+Additionally, there are [over two hundred ports of popular open-source software](Ports/AvailablePorts.md), including games, compilers, Unix tools, multimedia apps and more.
 
 ## How do I read the documentation?
 
 Man pages are available online at [man.serenityos.org](https://man.serenityos.org). These pages are generated from the Markdown source files in [`Base/usr/share/man`](https://github.com/SerenityOS/serenity/tree/master/Base/usr/share/man) and updated automatically.
 
 When running SerenityOS you can use `man` for the terminal interface, or `help` for the GUI.
 
-## How do I build and run this?
-
-See the [SerenityOS build instructions](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md)
-
-## Before opening an issue
+Code-related documentation can be found in the [documentation](Documentation/) folder.
 
-Please see the [issue policy](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#issue-policy).
+## How do I build and run this?
 
-FAQ: [Frequently Asked Questions](https://github.com/SerenityOS/serenity/blob/master/Documentation/FAQ.md)
+See the [SerenityOS build instructions](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md). Serenity runs on Linux, macOS (aarch64 might be a challenge), Windows (with WSL2) and many other *Nixes with hardware or software virtualization.
 
-## Get in touch
+## Get in touch and participate!
 
 Join our Discord server: [SerenityOS Discord](https://discord.gg/serenityos)
 
-## Author
+Before opening an issue, please see the [issue policy](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#issue-policy).
 
-* **Andreas Kling** - [awesomekling](https://twitter.com/awesomekling)
+A general guide for contributing can be found in [`CONTRIBUTING.md`](CONTRIBUTING.md).
 
-## Contributors
+## Authors
 
+* **Andreas Kling** - [awesomekling](https://twitter.com/awesomekling)
```

Comment:
Is it a desire of Andreas to be demoted from author + contributors to first among many authors? Seems to make it less clear that he's the BDFL of the project, which I believe still holds true?

Reply:
It reflects the current content of https://serenityos.org, also in a recent office hour live stream I asked him about basically that: https://www.youtube.com/watch?t=6046&v=e1_Lr2BCSMI